<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Audio output</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="audioAlsa">
          <property name="text">
           <string>Alsa</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="audioJack">
          <property name="text">
           <string>Jack</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="audioOss">
          <property name="text">
           <string>Oss</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="audioNull">
          <property name="text">
           <string>Null</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Midi input</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="midiAlsa">
          <property name="text">
           <string>Alsa</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="midiJack">
          <property name="text">
           <string>Jack</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="midiOss">
          <property name="text">
           <string>Oss</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="midiNull">
          <property name="text">
           <string>Null</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>150</width>
            <height>53</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAudio_changed">
   <property name="text">
    <string>Audio changed</string>
   </property>
  </action>
  <action name="actionMidi_changed">
   <property name="text">
    <string>Midi changed</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>audioAlsa</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionAudio_changed</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>audioJack</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionAudio_changed</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>audioNull</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionAudio_changed</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>audioOss</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionAudio_changed</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>midiJack</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionMidi_changed</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>midiOss</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionMidi_changed</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>midiNull</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionMidi_changed</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>midiAlsa</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionMidi_changed</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
